apps/boilerplate-front-end/src/Main.jsx

```diff
@@ -160,6 +179,12 @@ const Main = () => {
     }
   }, [])
 
+  useEffect(() => {
```

Comment:
Comment to explain? Not sure what's going on here, and will this work in prod?

Reply:
It's to be sure than if we open the app and are on 127...... and not on Logalhost it will change the adress because the OKTA admin are very strict and they're waiting exaclty the adress http://localhost:5173 and the adress of deployed one

Reply:
ok understood, thank you!